core/controllers/feedback.py

```diff
@@ -158,7 +158,7 @@ class SuggestionActionHandler(base.BaseHandler):
     _ACCEPT_ACTION = 'accept'
     _REJECT_ACTION = 'reject'
 
-    @editor.require_editor
+    @acl_decorators.can_manage_suggestions_on_exploration
```

Comment:
I see this as basically being the same as "can edit exploration" (that's what the "suggestion manager" is doing).